kotlinpoet/src/main/java/com/squareup/kotlinpoet/AnnotationSpec.kt

```diff
@@ -30,7 +30,13 @@ class AnnotationSpec private constructor(
   builder: AnnotationSpec.Builder,
   private val tagMap: TagMap = builder.buildTagMap()
 ) : Taggable by tagMap {
-  val className: ClassName = builder.className
+  @Deprecated(
```

Comment:
Error?

Reply:
I think warning is fine, there's no immediate need to change use sites upon upgrading, `className` would continue to work until you start creating parameterized `AnnotationSpec`s.